go/src/socialapi/workers/countly/api/api.go

```diff
@@ -0,0 +1,123 @@
+package api
+
+import (
+	"koding/db/mongodb/modelhelper"
+	"koding/tools/utils"
+	"net/http"
+	"net/url"
+	"socialapi/config"
+	"socialapi/models"
+	"socialapi/workers/common/response"
+	"socialapi/workers/countly/client"
```

Comment:
Yes, that's good idea.

I _think_ it'd be even possible to improve the heuristics, to mark a package as local and group it accordingly if it exists in the same GOPATH as the source file that is being processed. I haven't read the source code of the goimports thought.